zk_toolbox/crates/config/src/wallets.rs

```diff
@@ -16,6 +16,10 @@ pub struct WalletsConfig {
     pub fee_account: Wallet,
     pub governor: Wallet,
     pub token_multiplier_setter: Wallet,
+    /// A developer multisig account exists on mainnet.
+    /// The intention here is to use its address to own certain contracts,
+    /// while the private key is either empty, or different for each member.
+    pub developer_multisig: Option<Wallet>,
```

Comment:
I don't think we have to implement it here. We could setup it better manually

Reply:
Okay, but where do you propose to set it manually? 

First I thought that it could be set as an argument to the `deploy-consensus-registry` command, but then it wouldn't have a value if we just go with `deploy-l2-contracts` which does it all and gets all the parameters from the config, and the `governor` is even the preferred value on testnet. Then it automatically writes the config for the forge scripts and executes, so there is no more time to manually set it there.

Reply:
Let's add a command for changing it. 
set it by default to governor and add an additional step to change it

Reply:
Can you clarify what exactly the command would change? Do you mean to add a method to `DeployContract.sol` to change the owner of the contract after deployment, and a command to the `zk_inception` tool to invoke it with a new address?

Reply:
I just had a look at how ownership transfer is supposed to work. 

As I understand I can't call the [initialize](https://github.com/matter-labs/era-contracts/blob/446d391d34bdb48255d5f8fef8a8248925fc98b9/l2-contracts/contracts/ConsensusRegistry.sol#L39) function again, which would immediately transfer the ownership from the governor to the dev multisig, because it's already been initialised when the proxy got deployed. 

Instead I would have to call the public [transferOwnership](https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/22489db15621b9a42ebddb1facade6962034e9b9/contracts/access/Ownable2StepUpgradeable.sol#L60) function which would start a 2-step transfer to the dev multisig, and then follow up by the [acceptOwnership](https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/22489db15621b9a42ebddb1facade6962034e9b9/contracts/access/Ownable2StepUpgradeable.sol#L79) signed by the multisig to finish the procedure. 

Do you propose adding support to the `zk_toolbox` to with this multiple step process? 
1. If the `governor` wallet is the current owner of the registry, sign a transaction to initiate the transfer
2. I don't know how the Ganache multisig works, but another command would be executed by developers who have access to the private keys in the multisig - it seems a bit overly specific to the use case to have a command for this purpose

Reply:
I think we could introduce the function with only transfer ownership